### ISSUE TITLE
Cyrillic te has too much bounding space

### ISSUE THREAD

**Issue by Ledokol** [source-foundry/Hack#431]
_т_  creates visible gaps.

![source](https://user-images.githubusercontent.com/6161120/40578953-f26fbe8a-6136-11e8-869f-869b431499ad.png)
![plus60](https://user-images.githubusercontent.com/6161120/40578955-f5355756-6136-11e8-93de-0aafeae46321.png)
![80](https://user-images.githubusercontent.com/6161120/40579151-bc69596e-613a-11e8-834f-f56367061b3f.png)

What do you think about adjusting it's width?

**Comment by chrissimpkins** [comment]
Thanks for the report Pavel.  Do you suggest that we extend the horizontal stroke so that it is wider on both sides?  Is this work that you would like to do and submit as a PR?

**Comment by Ledokol** [comment]
Yep, if extended horizontal stroke looks fine, I will prepare PR for all fonts of a typeface.

**Comment by chrissimpkins** [comment]
Sounds great! Look forward to it.

**Comment by chrissimpkins** [comment]
Fixed by @Ledokol and merged to `dev` branch.  These changes will be included in the v4.000 release.
